apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
I just re-copied HelloHexagon with Andrew's modernization (thanks for doing that!) and reimplemented the matrix multiply.